src/prop/model.py

```diff
@@ -136,6 +137,8 @@ def PropagationLoss(self, f, lat1, lng1, h1, lat2, lng2, h2, land_cat=''):
       
     if land_cat == 'RURAL' or h1 >= 200 or h2 >= 200:
       itm_loss = self.ITM_AdjustedPropagationLoss(lat1, lng1, h1, lat2, lng2, h2, f, 0.5)
+      if land_cat == 'RURAL':
```

Comment:
Okeydokey. Although, we don't actually set other clutter losses to zero, they're built deterministically into eHata, but I agree, if we add random losses it's impossible to compare on a point-to-point basis.